src/conf_mode/interfaces-tinc.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import shutil
+
+from sys import exit
+
+from netifaces import interfaces
+from vyos.configdict import get_interface_dict,leaf_node_changed
+from vyos.config import Config
+from time import sleep
+from vyos.ifconfig import TincIf
+from vyos import ConfigError
+from vyos.util import call
+from vyos.template import render
+from vyos.configverify import verify_address
+from vyos.configverify import verify_dhcpv6
+from vyos.configverify import verify_vrf
+
+from vyos import airbag
+
+airbag.enable()
+
+def get_config(config = None):
+    if config:
+        conf = config
+    else:
+        conf = Config()
+        
+    base = ['interfaces', 'tinc']
+    
+    tinc = get_interface_dict(conf, base)
```

Comment:
must be `_, tinc = get_interface_dict(conf, base)`

Reply:
Seems to be lost in late conflict resolution, I've manually restored this and kept the mainline in sync